lib/smart_answer/calculators/self_assessment_penalties.rb

```diff
@@ -8,18 +8,21 @@ class SelfAssessmentPenalties < OpenStruct
         "2012-13": Date.new(2014, 1, 31),
         "2013-14": Date.new(2015, 1, 31),
         "2014-15": Date.new(2016, 1, 31),
+        "2015-16": Date.new(2017, 1, 31),
```

Comment:
It might be useful for incoming tests to mimic these values in: https://github.com/alphagov/smart-answers/blob/66b9339fde188b3f383e7cdc0eb528fc561cadc2/test/unit/calculators/self_assessment_penalties_test.rb#L6-L22